pinot-common/pom.xml

```diff
@@ -301,6 +303,10 @@
       <artifactId>equalsverifier</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
this makes it a project dependency, while it is only used by the plugin. According to https://maven.apache.org/plugins/maven-antrun-plugin/examples/customTasks.html you should add the dependency to the plugin .